### DISCUSSION TITLE
DDF units

### DISCUSSION THREAD

**Discussion by Andjelaaaa** [DeepRegNet/DeepReg#833] [Q&A] [answered]
Hi,

Your toolkit is great! I was just wondering, is the outputted/predicted DDF in pixel values? Hence, to obtain the mm values, it would have to be converted on our end. Is this right?

Thank you very much!

**Comment by mathpluscode** [comment, marked as answer]
Hi Andjelaaaa,

Thanks for your message and sorry for the late response. Yes, this is correct, the DDF/DVF units are per pixel/voxel.

Bests.
